testing/rdedup/APKBUILD

```diff
@@ -5,7 +5,7 @@ pkgver="3.1.1"
 pkgrel=1
 pkgdesc="Data deduplication with compression and public key encryption"
 url="https://github.com/dpc/rdedup"
-arch="x86_64 armhf armv7" # limited by cargo
+arch="x86_64 armv7 armhf aarch64 !x86 ppc64le" # limited by cargo (!x86 due linking error in cc package)
```

Comment:
I guess the -fno-stack-protector thingie could fix that too

Reply:
Done